### DISCUSSION TITLE
More info to display on a standalone page

### DISCUSSION THREAD

**Discussion by Arlequ1n** [owncast/owncast#948] [Q&A]
Hello there !

I discovered owncast a few days ago and i LOVE it. Already been streaming a lil bit of video games and talks and stuff, so neat.

I have a question though : is there a simple way to get the streaming infos like the viewer count, the used processor on my VPS, stuff like that, on an HTML view ? The goal is to display it in a dock on OBS so i don't have to watch on the admin page all the times.
I found where i can have those infos in the API but before coding stuff and putting my hands in there i wanted to know if someone figured out a way to do that.

Thanks in advance !

**Comment by gabek** [comment]
I'm glad you're enjoying Owncast, thanks for checking it out!

I don't think anybody has built any external admin UI before, but it sounds like a really cool idea!  I didn't even know you could embed HTML into OBS, that's neat.  Feel free to reach out if there's any data you need that isn't currently covered by the existing admin APIs.
